src/app/game/schulte-table/page.tsx

```
@@ -0,0 +1,156 @@
+          </button>
+        ))}
+      </div>
+
+      {/* Popup when game ends */}
+      {showPopup && (
+        <div className={styles.popupOverlay}>
+          <div className={styles.popup}>
+            <h2>🎉 Game Completed!</h2>
+            <p>Your time: <strong>{elapsedTime.toFixed(1)}s</strong></p>
+            <button className={styles.restartBtn} onClick={startNewGame}>
+              Restart Game
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)